index.html

```diff
@@ -4,19 +4,51 @@
   <meta charset="UTF-8">
   <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"/>
   <title>판다마켓</title>
-  <link rel="stylesheet" href="sprint2.css">
-  <link rel="icon" href="img/logopanda.svg">
+  <link rel="stylesheet" href="./css/index.css">
+  <link rel="icon" href="./img/pandaLogoIcon.svg">
+<!--   
+    <meta property="og:title" content="Panda Market" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="./img/webshop.png" />
+    <meta property="og:url" content="https://endearing-fairy-27f4f6.netlify.app/" />
+    <meta property="og:description" content="일상의 모든 물건을 거래해보세요" />
+    <meta property="og:site_name" content="판다 마켓" />  
+  
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:title" content="Panda Market" />
+    <meta name="twitter:description" content="일상의 모든 물건을 거래해보세요" />
+    <meta name="twitter:image" content="./img/webshop.png" />
+    <meta name="twitter:url" content="https://endearing-fairy-27f4f6.netlify.app/" /> -->
+
+<!-- Facebook Meta Tags -->
+<meta property="og:url" content="https://67d15a7ccce1f07de4e97ae7--transcendent-dragon-e136c7.netlify.app/">
+<meta property="og:type" content="website">
+<meta property="og:title" content="판다 마켓">
+<meta property="og:description" content="일상의 모든 물건을 거래해보세요">
+<meta property="og:image" content="https://opengraph.b-cdn.net/production/images/34772fdc-0142-45bd-84fe-2943632d1fc5.jpg?token=HEFF0d59Tdf_G9HN0zY2MHiDUBnM0HtT8FtkMmG4XsQ&height=630&width=1200&expires=33277773538">
```

Comment:
❗️ 수정요청
해당 위치에는 원규님이 올려주신 미리보기 이미지 주소가 들어가야 합니다~
해당 사항을 배포하시고 아래처럼 설정하시는 것을 추천드려요~
참고하셔서 트위터 관련 meta 태그도 변경해보세요!

```suggestion
<meta property="og:image" content="https://pandanara99.netlify.app/img/webshop.png">
```